smartsim/_core/_install/builder.py

```diff
@@ -39,9 +39,15 @@ def __init__(self, env, jobs=1, verbose=False):
 
         # Find _core directory and set up paths
         _core_dir = Path(os.path.abspath(__file__)).parent.parent
+
+        dependency_path = _core_dir
+        if os.getenv('SMARTSIM_DEP_PATH'):
```

Comment:
I think you ended up calling this ``SMARTSIM_DEP_INSTALL_PATH``

Reply:
There's the `SMARTSIM_DEP_INSTALL_PATH` which is set in the builder's environment. On the user side of it, the environment modules set `SMARTSIM_DEP_PATH`.

Reply:
Oh yeah, I remember why I called them different things. It's because of the way that environment variables set things. If I made it consistent we would incorrectly replace lines like:
```
set          smartsim_dep_path SMARTSIM_DEP_INSTALL_PATH

prepend-path LD_LIBRARY_PATH   $smartsim_dep_path/lib
prepend-path PATH              $smartsim_dep_path/bin

setenv       SMARTSIM_DEP_PATH SMARTSIM_DEP_INSTALL_PATH
```
While I could write some extra scripting to go around that, IMO it seems more reasonable to keep this the way it is. The rationale is that `SMARTSIM_DEP_INSTALL_PATH` is for a specific compiler chain, whereas `SMARTSIM_DEP_PATH` is the generic environment variable which can be changed by an environment module.